package.json

```diff
@@ -1,6 +1,6 @@
 {
   "name": "ember-router-scroll",
-  "version": "0.6.0",
+  "version": "0.7.0",
```

Comment:
Why are we releasing a new version just to add changelogs?

Reply:
No. It's not a feature that is shipped to users, so users will think 0.7.0 offers them something, and it does not.

Reply:
Not only is this not a new feature from the perspective of the published package, but this version bump also marks it as a _breaking_ change. You're signaling to anyone using this addon that they need to come do research to understand what's different before they can safely upgrade.

Reply:
I will get feedback from the people I requested code review from. They may agree with you @rwwagner90 . 😊

Reply:
I consulted with some teammates. They suggested removing the CHANGELOG.md all together. That would mean the `chore` scripts and the lerna-changelog in this PR would be removed but the `postpublish` would remain. 

This would:
- resolve the [bug—now cleanup](https://github.com/dollarshaveclub/ember-router-scroll/commit/8d885440d6a6de71d6ccde830c8c6fdb30114a46#r28387726)
- close #103 
- would only be a patch. 

@snewcomer I'd appreciate your thoughts. 😊

Reply:
Unless I am misinformed about semver, we don't need to change the version at all here or release anything. It's just manipulating scripts and changelogs for people developing the addon, not for consumers of the addon. Changing the version of a package signals to the consumers that something changed.

Reply:
That being said, I do like changelogs, and I am fine with scripts to update them, I just don't think we want to change the version of this package.

Reply:
Okay, so the people communicating in this PR want this PR but with **no** npm version bump?

----

### Future note

If this addon uses CircleCi (which will allow for more community support), npm publishing can be automated...but will need [semver version bumps](https://docs.npmjs.com/getting-started/semantic-versioning) to publish w/o failure. 

That said, my team has built a script to ignore files that are deemed **"harmless"** within a repo—[harmelss changes](https://github.com/dollarshaveclub/harmless-changes). This package could be implemented with a `.ciignore` file to not fail CI for like, `.md` files without a semver version bump. 

Even after implementing Harmless Changes, `package.json` updates **can be** harmful—so `package.json` updates would require a version bump...if this addon starts to leverage automation opportunities which I hope people aren't against.

Reply:
I just don't think we should version things that are not versions. Once these processes are in place, all the changes should be actual functional ones, but might not always be. If changes are made that do not effect the consumer and do not change functionality, we should allow for a way to not version.